spanner/api/Spanner.Samples/CreateDatabaseAsyncPostgre.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_postgresql_create_database]
+
+using Google.Cloud.Spanner.Admin.Database.V1;
+using Google.Cloud.Spanner.Common.V1;
+using System;
+using System.Threading.Tasks;
+
+public class CreateDatabaseAsyncPostgreSample
+{
+    public async Task CreateDatabaseAsyncPostgre(string projectId, string instanceId, string databaseId)
+    {
+        DatabaseAdminClient databaseAdminClient = await new DatabaseAdminClientBuilder().BuildAsync();
+
+        // Create the CreateDatabaseRequest with PostgreSQL dialect and execute it.
+        // There cannot be Extra DDL statements while creating PostgreSQL.
+        var createDatabaseRequest = new CreateDatabaseRequest
+        {
+            ParentAsInstanceName = InstanceName.FromProjectInstance(projectId, instanceId),
+            CreateStatement = $"CREATE DATABASE \"{databaseId}\"",
+            DatabaseDialect = DatabaseDialect.Postgresql
+        };
+
+        var createOperation = await databaseAdminClient.CreateDatabaseAsync(createDatabaseRequest);
+
+        // Wait until the operation has finished.
+        Console.WriteLine("Waiting for the operation to finish.");
```

Comment:
This is just a nit, but when you run this sample this text will be printed twice, once for creating the db and once for adding the tables. Consider distinguising the two as in "Waiting for the DB to be created" and "Waiting for the tables to be created."

Here and everywhere where there's something like this happening.

Reply:
Done.